pkg/kubelet/kubelet_pods.go

```diff
@@ -185,7 +185,10 @@ func makeMounts(pod *v1.Pod, podDir string, container *v1.Container, hostName, h
 				return nil, cleanupAction, fmt.Errorf("volume subpaths are disabled")
 			}
 
-			if filepath.IsAbs(subPath) {
+			// On Windows, filepath.IsAbs does not consider \ or / prefixed paths as absolute, even though
```

Comment:
I think this check is also performed at spec validation which should currently only ever happen on code compiled for linux.

https://github.com/kubernetes/kubernetes/blob/544b74c2cb94fce6d89019b85c7c891050df2af7/pkg/apis/core/validation/validation.go#L2427-L2430

calls

https://github.com/kubernetes/kubernetes/blob/544b74c2cb94fce6d89019b85c7c891050df2af7/pkg/apis/core/validation/validation.go#L1158-L1170